test/test_fcntl.rs

```diff
@@ -1,26 +1,27 @@
-use nix::Error;
 use nix::errno::*;
-use nix::fcntl::{openat, open, OFlag, readlink, readlinkat, renameat};
+use nix::fcntl::{open, openat, readlink, readlinkat, renameat, OFlag};
 use nix::sys::stat::Mode;
 use nix::unistd::{close, read};
-use tempfile::{self, NamedTempFile};
+use nix::Error;
 use std::fs::File;
 use std::io::prelude::*;
 use std::os::unix::fs;
+use tempfile::{self, NamedTempFile};
 
 #[test]
 fn test_openat() {
     const CONTENTS: &[u8] = b"abcd";
     let mut tmp = NamedTempFile::new().unwrap();
     tmp.write_all(CONTENTS).unwrap();
 
-    let dirfd = open(tmp.path().parent().unwrap(),
-                     OFlag::empty(),
-                     Mode::empty()).unwrap();
-    let fd = openat(dirfd,
-                    tmp.path().file_name().unwrap(),
-                    OFlag::O_RDONLY,
-                    Mode::empty()).unwrap();
+    let dirfd = open(tmp.path().parent().unwrap(), OFlag::empty(), Mode::empty()).unwrap();
```

Comment:
It looks like you made some formatting changes unrelated to this PR.  Could you please back those out?

Reply:
Sorry, forgot to disable rustfmt while editing. It's fixed.